cli/runtime/plugin/usage.go

```diff
@@ -23,6 +24,12 @@ var UsageFunc = func(c *cobra.Command) error {
 	return t.Execute(os.Stdout, c)
 }
 
+var au aurora.Aurora
+
+func init() {
+	au = aurora.NewAurora(!component.IsTTYEnabled())
+}
```

Comment:
To avoid having this code duplicated, I wonder if we could move it to `component` along with the 5 functions at the bottom, make the functions public and use them in `cli/core/pkg/cli/usage.go`?